framework/include/meshgenerators/BreakMeshByBlockGenerator.h

```diff
@@ -26,7 +27,14 @@ class BreakMeshByBlockGenerator : public BreakMeshByBlockGeneratorBase
   std::unique_ptr<MeshBase> generate() override;
 
 protected:
+  subdomain_id_type blockRestricteElementSubdomainID(const Elem * elem);
```

Comment:
```suggestion
  subdomain_id_type blockRestrictedElementSubdomainID(const Elem * elem);
```

Can you add a doxygen style comment as to what this does and why it is needed?